core/src/test/scala/wartremover/contrib/warts/NoNeedForMonadTest.scala

```diff
@@ -7,27 +7,29 @@ import org.wartremover.test.WartTestTraverser
 
 class NoNeedForMonadTest extends FunSuite with ResultAssertions {
 
-  val message = "[wartremover:NoNeedForMonad] " + NoNeedForMonad.message
-
   test("Report cases where Applicative is enough") {
     val withWarnings = WartTestTraverser(NoNeedForMonad) {
       for {
         x <- List(1, 2, 3)
         y <- List(2, 3, 4)
-      } yield x * y
+      } yield {
+        x * y
+      }
```

Comment:
I believe all the similar following changes are style-changes, and either need to be put in a separate commit or undone altogether

Reply:
Please revert unnecessary style changes :pray: